packages/webdriverio/src/utils.js

```diff
@@ -77,13 +77,6 @@ export const findStrategy = function (value, isW3C, isMobile) {
         using = 'tag name'
         value = value.replace(/<|>|\/|\s/g, '')
 
-    // use name strategy if value queries elements with name attributes
-    // e.g. "[name='myName']" or '[name="myName"]'
-    } else if (value.search(/^\[name=("|')([a-zA-z0-9\-_. ]+)("|')]$/) >= 0) {
```

Comment:
The name selector is still valid for JSONWP, so we should just add 
```js
 } else if (!isW3C && value.search(/^\[name=("|')([a-zA-z0-9\-_. ]+)("|')]$/) >= 0) {
```

Reply:
Since the back end just converts the name strategy to css... is there even a point to this? Why not just send it as CSS and pretend that strategy doesn't exist?

Reply:
> Since the back end just converts the name strategy to css

I don't think the driver converts anything.

> Why not just send it as CSS and pretend that strategy doesn't exist?

I think the name selector strategy is used for not only web elements. It can have special meaning. So we should provide a way to use it.